pyproject.toml

```diff
@@ -17,6 +17,7 @@ line_length = 88
 multi_line_output = 3
 use_parentheses = true
 skip_glob = [".direnv", "*migrations*", "tmp", ".venv"]
+known_third_party = ["crispy_forms", "django", "fabric", "hypothesis", "pytest", "pytest_django", "sentry_sdk"]
```

Comment:
What's this for?

Reply:
It's seeded by [one of configured pre-commit tools](https://github.com/asottile/seed-isort-config) - when it finds an import of a module not in the project or stdlib it seeds that isort option.  It's an extra level of stability for isort which I've seen trip up on some files from time to time.